dev-lang/gnuprologjava/gnuprologjava-0.2.6-r1.ebuild

```diff
@@ -15,16 +15,13 @@ KEYWORDS="amd64 x86"
 IUSE="doc"
 
 BDEPEND="app-arch/unzip"
-RDEPEND=">=virtual/jdk-1.6:="
+RDEPEND=">=virtual/jdk-1.8:="
```

Comment:
needs a revbump as it changes content of `package.env`. also `=` should be replaced with `*`.